data/nmstate/003-nmstated.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
```

Comment:
```
message: 'could not apply (apps/v1, Kind=DaemonSet) nmstate/nm-state-controller:
could not create (apps/v1, Kind=DaemonSet) nmstate/nm-state-controller: DaemonSet.apps
"nm-state-controller" is invalid: spec.template.metadata.labels: Invalid value:
map[string]string{"nmstate.io":"state-controller"}: `selector` does not match
template `labels`'
```

With the new version you should add the same label in the spec section

Reply:
Oh yeah, you are right. Thanks

Reply:
Fixed, could you please take another look?